code/game/objects/effects/landmarks.dm

```diff
@@ -40,6 +40,14 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/newplayer_start) //Without this you sp
 	name = "traderstart_sol"
```

Comment:
Could probably rename this landmark name/path since its not a sol exclusive thing anymore